runtime/vm/rv64cinterpnojit.cpp

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "BytecodeInterpreter.hpp"
+#include "ut_j9vm.h"
+
+/* This code here should be removed once the JIT related code
+ * is implemented in rv64cinterp.m4.
+ */
+extern "C" {
+
+void
```

Comment:
This function should be removed - since you have the wrapper m4 file, you may as well declare the symbol there.

Reply:
@gacholio , the mp4 file (not working for now) is an empty file as a placeholder to be modified by JIT to add JIT related assembly in there.

Reply:
As discussed with @gacholio offline, vm->bytecodeLoop(currentThread) should be directly called via c_cInterpreter while cInterpreter is moved to the .mp4 file to be added along with JIT code there.